src/middlewared/middlewared/plugins/pool.py

```diff
@@ -2941,7 +2941,7 @@ def transform(dataset):
         ]), has_default=False),
         Inheritable(Str('casesensitivity', enum=['SENSITIVE', 'INSENSITIVE', 'MIXED']), has_default=False),
         Inheritable(Str('aclmode', enum=['PASSTHROUGH', 'RESTRICTED', 'DISCARD']), has_default=False),
-        Inheritable(Str('acltype', enum=['OFF', 'NOACL', 'NFSV4', 'NFS4ACL', 'POSIX', 'POSIXACL']), has_default=False),
+        Inheritable(Str('acltype', enum=['OFF', 'NFSV4', 'POSIX']), has_default=False),
```

Comment:
Are their usages still valid in `acl_linux.py` ?

Reply:
Sorry. Forgot about question. No, these aren't used in acl_linux.py.